cleverhans_tutorials/mnist_tutorial_deepfool.py

```diff
@@ -0,0 +1,184 @@
+# adapted from mnist_tutorial_cw.py
```

Comment:
Can you remove this tutorial? (see the discussion we had on the previous PR #260)

Reply:
we have deleted those tutorials.